Adyen.podspec

```diff
@@ -12,10 +12,14 @@ Pod::Spec.new do |s|
   s.source = { :git => 'https://github.com/Adyen/adyen-ios.git', :tag => "#{s.version}" }
   s.platform = :ios
   s.ios.deployment_target = '12.0'
-  s.swift_version = '5.7'
+  s.swift_version = '5.9'
   s.frameworks = 'Foundation'
-  s.default_subspecs = 'Core', 'Components', 'Actions', 'Card', 'Encryption', 'DropIn', 'Session'
-  s.pod_target_xcconfig = {'SWIFT_SUPPRESS_WARNINGS' => 'YES' }
+  s.default_subspecs = 'Core', 'Components', 'Actions', 'Card', 'Encryption', 'DropIn', 'Session', 'AdyenCheckout'
```

Comment:
`AdyenCheckout` has all of them as dependencies, can we remove everything but `AdyenCheckout`?

Reply:
It may not work with cocoapods as DropIn was the same and it also expliclitly states all the dependencies.